templates/rhpam78-authoring-ha.yaml

```diff
@@ -937,8 +937,6 @@ objects:
           - name: KIE_MBEANS
             value: "${KIE_MBEANS}"
 ## OpenShift Enhancement BEGIN
-          - name: KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED
```

Comment:
@desmax74 We can not remove this environment variable, rather need to change the value from 'false' to 'true'. The reason behind it is that by turning that switch on it will allow Business Contral console to be able to discovery KIE Server instances with 'OpenShiftStartupStrategy'. So those two env variables must be in pairs as the following:
 

- "KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED" = "true" <=> "KIE_SERVER_STARTUP_STRATEGY" = "OpenShiftStartupStrategy" 
- "KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED" = "false" <=> "KIE_SERVER_STARTUP_STRATEGY" = "ControllerBasedStartupStrategy"